### PR TITLE
UCS/DEBUG: Make ucs_debug_get_lib_path() thread safe

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11204/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 9 of 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 19-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/444abcf45dc176611a1bf4f05aab9cfb5410470a/_main) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/31225480f589be35a7fe81da1553f0807e3190ed/hpc-arm-cavium-jenkins_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/351ed1abd6e8fc1825aa287ce186004f71390fc0/hpc-arm-cavium-jenkins_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/55f8bfe848d91f80c95c303774fd4617b617db4d/hpc-arm-cavium-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/a7ecbffcb91cd5ed3e1aa6a7be4fe5e6c38ea970/hpc-arm-cavium-jenkins_W3) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/a06c83662bf20edf7ed7a33d825896676a97c598/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/994dc708bd71a926024e4e8dd8afcf03d8bf1fd8/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/11e312a5bf6947f69a29fa95354368ce7b87c91e/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/13592dff598f078a97634825195610f11ef21569/hpc-arm-hwi-jenkins_W3) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/b65678dc15f0499ddf83cf1f9e0747784d55fe95/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/d5790bb7a759922cc28a55cc77f2b8699ccf886a/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/2f428bd1ea349570fd280e5f739e8b2e9ad9d202/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/936e197048aad399aeafd3c8d67871577a3db93d/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/5ad6c810d07e2b74b9d68aa1c98b58006aa8e4bd/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/14f0fd585215c0f8e2004ed6126b450636fa5bcf/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/922e4a2df78c2455d61cc0112994eacac4a47f22/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/a40b1472b3c6a6e14ec7784da000329922d664b6/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/ed498dabaaeaeef6b242c6bd12e1e63b2fbae315/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/771453f507a6c69812c3c3f5f71094dcbb144275/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/bd0fa5c27851caa1952a0b5de3da4593a609f964/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/e1df3398ff4a7afea69e553002f44ec57db5246f/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/a9265e05c6ffe687d89a6e5cfd71336cde4ee6ae/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/3da7ca2e7f3532dbbe75bc63cfbb9992e87a986b/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/a8b27c764ca44c1606b351eef31bccefbc6ac090/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/ee7c22131877c984f0deb1797fe23f5c00327216/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/7b8f228e0712b6d32bfa4b0339eaaa4e8ca62d79/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/7839f100cd55e24884742c8231c7faab03e03a17/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/02743c32d6789bc239928ef33f19b41f09746ccf/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7b4746b8fdea9d5dc5b789041ec180b6/raw/c9b023a39453c65f1c6e8f72c1082822be9b31f2/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11221/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 9 of 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 20-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/a3182553b289fab26dc185af8acfaa0be8184c27/_main) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/0dce1ec29de1fcf1cb6f539d62031edd5209114a/hpc-arm-cavium-jenkins_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/e2826dcb500d49c88b7cf90b585df4f7f8230ea2/hpc-arm-cavium-jenkins_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/e410e110ac1e528a13e4ade34d697028bef2fcc3/hpc-arm-cavium-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/818fe837dd3d9bb82889224812bd7ad595ce88c9/hpc-arm-cavium-jenkins_W3) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/bd689333e50fb4517cef37b0d7e6e3c3bf00e186/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/cd5902b5ac15a28746934168f507a684e8bba406/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/8094ba9a847f51c3e8686d8f3f9a99b963b1a59f/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/f338698b9f8fdbba72c4af106e1cefc22b3825ac/hpc-arm-hwi-jenkins_W3) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/539c9b95bd7ee66a9387b0716f2bc0a6cdf2405f/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/982fd91a3add52c76f6b4b9a8cfb60aea9e36115/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/0e4e8c227814b866cacbc7d40b1363416ee6f851/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/d3027da0426ff0ae45aec66a0e6ba11a960748ca/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/66451ec7e45030b81e7bcdf3e24268337f682d51/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/1573354c7bbffd6fa9abbc185446403df21c1ed9/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/e311aa7007cff5c18132e08b30a713dff1049c41/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/211140c1e8eff52b4efe86dd633d370b451f1501/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/8e23a8b99e0b8939979972d48e8ab9e00a5c3db6/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/a7deca69f189ba2ba09c2841931de423e394cc8c/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/9da0bea6d6bb1b2eca462e9c72947aeb6aa94b32/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/faff227455436b7349218e93443095f43662569a/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/84c5bdfd01cbf3397e72b8edba5f29ca0fd972ad/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/6abe02625d28ca8214c197ecb68cde65e061d3ce/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/eccd21d1d91efed6d6812e197358ed22753257e1/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/b07d6c09765383bea86ff4786a7ff0896aa9111a/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/2543df78c54bcb89a6f3601b6cc3c96b85bdbd0a/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/d4f0bde1164939ad08306d374c47ae19b4bd2a47/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/315becc25df5bb222d511e7695d8e6d5b0dfa66a/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/afd21a80684a00c435c575ff6519ebe5/raw/3cecd7040dc1de35c3207b06670088ac33b5955a/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/ucs/debug/debug.c [hunk 1]
```diff
@@ -156,6 +156,8 @@ static int dl_match_address(struct dl_phdr_info *info, size_t size, void *data)
     return 0;
 }
 
+#ifdef HAVE_DETAILED_BACKTRACE
```
Comment (dmitrygladkov) [inline src/ucs/debug/debug.c]: move `#ifdef HAVE_DETAILED_BACKTRACE` before `dl_match_address` function declaration to fix compiler warning from testing?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11228/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 20-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/969755132600125444de8372a101644004268504/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/5cdbe84dbef5bed3500d27f06d851c39d040b3aa/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/a693d71ffd0a8b402fb249e61719f406af670d64/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/0d3b97095e0a57befb72bad30c04638e8a6d0130/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/f45beb3578e59d8de9ba6122109880c34f628567/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/958d2df2421956a11caa74fea222a2d57a4e8e0c/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/f9dc05e9c1a6be4fe6b46e8c26e16c75a1776e4b/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/f58d87c1a28e162bb6deacafbce417e5fc2dad45/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/5689e757fdc1f84d66c1128b0649777225578635/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/426a0df6f8ffe1a31350cb2b084aa7b3277caba5/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/1bbc2fb32fb0779dbfc7dd0b9749b8ea3657854b/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/513057eeef2bc0f572f5c5625a9c7fe9f4b54e39/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/40274911b5d32cb98024b51c447924960d745178/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/42400eb88571747396250a398810f70168a18127/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/b50273bea7f25d6d88e94378067b834e2fa45229/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/42169edb6d5840f3a1a5a694859dc337e6a8da26/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/5fbccc006695293a0a9d7f21b2dc52705ac1b717/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/0fc4658dbf0f74afc7eb4b59696c86b2d5170070/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/4d87c22addac2cd4ddf29192e6736f1d0ff02461/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/69e504e72a51f2dd20fa9bb983250d6cac293638/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/9d490b0885cc0b4b121b9f2beadf7677920ac661/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/7fc9744f74cfdc62d57918e6569cb54b563b86e5/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/71f50d69d72ddde545146acff48d51f8fff90817/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/6c20dfe59f5f252cec1772fb4e1152707f8d7d16/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/c0b8ea40eb82c630922374248ec4eeec2d53e406/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/d93a5339613a89fe99561ce8b477ba13028fdc19/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/86b6cd5d2a7ef82665595371b4e6e20128c98b87/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/d68ecdbe6293733b56ffe02d2583d9ce50d1240c/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/54d6c56140d223de2095d82ba70f7951/raw/77799ee2521b147d77578ffa1612887d82a3f29f/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**